app/actions/EventActions.js

```diff
@@ -386,13 +386,17 @@ export function unfollow(followId: number, eventId: number): Thunk<Promise<*>> {
     );
 }
 
-export function isUserFollowing(eventId: number, userId: number) {
-  return callAPI({
-    types: Event.IS_USER_FOLLOWING,
-    endpoint: `/followers-event/?target=${eventId}&follower=${userId}`,
-    method: 'GET',
-    meta: {
-      errorMessage: 'Henting av hendelse følgere feilet'
-    }
-  });
+export function isUserFollowing(eventId: number): Thunk<Promise<*>> {
+  return (dispatch, getState) =>
+    dispatch(
+      callAPI({
+        types: Event.IS_USER_FOLLOWING,
+        endpoint: `/followers-event/?target=${eventId}&follower=${getState()
```

Comment:
My intuition still says that this potentially _can_ crash while in frontend. :fire:

The `state.auth.id` is used when selecting the user from the state inside the `EventDetailRoute`:
https://github.com/webkom/lego-webapp/blob/33c35bb32086df956818e675e74eca92ccda4a55/app/routes/app/AppRoute.js#L133

This code (also the old one) works flawlessly in SSR because of the waterfall-like execution of the actions. This is where the user is fetched in SSR: https://github.com/webkom/lego-webapp/blob/33c35bb32086df956818e675e74eca92ccda4a55/app/routes/app/AppRoute.js#L144

and this is where it is fetched during client side rendering: https://github.com/webkom/lego-webapp/blob/33c35bb32086df956818e675e74eca92ccda4a55/app/index.js#L54

When running on the client however, it will however fetch the actions inside `EventDetailRoute` right after this action (without depending on `User.FETCH`):

https://github.com/webkom/lego-webapp/blob/33c35bb32086df956818e675e74eca92ccda4a55/app/routes/app/AppRoute.js#L164


As seen here :arrow_down:, `User.FETCH.SUCESS` can potentially be dipatched after `Event.IS_USER_FOLLOWING.SUCCESS`; and `state.auth.id` isn't reduced into the state before `User.FETCH.SUCCESS`.
![screenshot from 2019-03-07 19-57-50](https://user-images.githubusercontent.com/1467188/53982082-46884d80-4114-11e9-98ca-4c451cd98de5.png)

The reason the fetch doesn't crash here, is because `/followers-event/?target=${eventId}&follower=` is a valid request.

**BUT**: The token is a jwt, so a hacky way is to run `jwtDecode(getState().auth.token).user_id` :fire: :fireworks: :fire_engine: :man_firefighter: :woman_firefighter: (or something like that). The best would however be a separate endpoint, or just `?follower=me`.